src/styles/index.css

```diff
@@ -0,0 +1,70 @@
+.stopwatch__circle,
+.stopwatch__second,
+.stopwatch__min,
+.stopwatch,
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@keyframes hand-rotate {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+body {
+  margin: 0;
+  height: 100vh;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  color: #000;
+  border: 1vmin;
+  border-style: dotted;
+  border-radius: 50%;
+}
```

Comment:
It's not clear why you are using `vmin` units for sizing the stopwatch. It may cause issues with responsiveness and scaling on different screen sizes. Consider using more predictable units like `px` or `rem`.